man/man1/zfs_volume_wait.1

```diff
@@ -0,0 +1,19 @@
+.Dd July 5, 2019
+.Dt ZFS_VOLUME_WAIT 1 SMM
+.Os Linux
+.Sh NAME
+.Nm zfs_volume_wait
+.Nd Wait for ZFS volume links in
+.Em /dev
+to be created.
+.Sh SYNOPSIS
+.Nm
+.Sh DESCRIPTION
+When a ZFS pool is imported, ZFS will register each ZFS volume
+(zvol) as a disk device with the system. As the disks are registered,
+.Xr \fBudev 7\fR
+will asynchronously create symlinks under /dev using the zvol's name.
```

Comment:
[nit] We might want to change `/dev` to be `/dev/zvol` specifically, since that seems to be the preferred link.